<launch>
  <node
    pkg="camera"
    type="webcam_pub.py"
    name="webcam_pub"
    output="screen"
  />
  <node
    pkg="camera"
    type="color_selection.py"
    name="color_selection"
    output="screen"
  />
</launch>
